File: Data Structures & Algorithms/duplicate-integer/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
       let map = new Set(nums)

        if(map.size !== nums.length) return true
        return false
    //    console.log(map)
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashmap = {}
        lt = []

        for i in range(len(nums)):
            final = target - nums[i]
            if final in hashmap:
                lt.append(hashmap[final])
                lt.append(i)
            hashmap[nums[i]] = i
        return lt